src/ceph-volume/ceph_volume/devices/lvm/common.py

```diff
@@ -15,30 +15,30 @@ def common_parser(prog, description):
     required_args = parser.add_argument_group('required arguments')
     parser.add_argument(
         '--journal',
-        help='A logical volume (vg_name/lv_name), or path to a device',
+        help='(filestore) A logical volume (vg_name/lv_name), or path to a device',
     )
     required_args.add_argument(
         '--data',
         required=True,
         type=arg_validators.LVPath(),
-        help='A logical volume (vg_name/lv_name) for OSD data',
+        help='OSD data path. Bluestore: A physical device or volume group. Filestore: A logical volume (vg_name/lv_name)',
     )
     parser.add_argument(
         '--journal-size',
         default=5,
         metavar='GB',
         type=int,
-        help='Size (in GB) A logical group name or a path to a logical volume',
+        help='(filestore) Size (in GB) A logical group name or a path to a logical volume',
```

Comment:
does ``--journal-size`` really accept a vg or path to an lv?

Reply:
Good catch, fixed.